packages/okta-angular/test/e2e/harness/src/app/app.component.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { TestBed, async, ComponentFixture } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { AppComponent } from './app.component';
+import { environment } from './../environments/environment';
+
+import {
+  OktaAuthGuard,
+  OktaAuthModule,
+  OktaCallbackComponent,
+  OktaLoginRedirectComponent
+} from '@okta/okta-angular';
+
+describe('Unit Tests', () => {
```

Comment:
These spec files should be siblings of the source files, it's odd that we're burying unit testing within an e2e test.  We should discuss this, we need to decide if we're going to fix this now or later.  I also need a place to put unit testing for the user agent work

Reply:
As discussed offline: this is going to be significant work and duplication of test boilerplate, so we won't do this.